Libraries/StyleSheet/StyleSheet.js

```diff
@@ -13,7 +13,16 @@
 
 var StyleSheetRegistry = require('StyleSheetRegistry');
 var StyleSheetValidation = require('StyleSheetValidation');
-var flattenStyle = require('flattenStyle');
+
+/* TODO(brentvatne) docs are needed for this */
+var flatten = require('flattenStyle');
+var PixelRatio = require('PixelRatio');
```

Comment:
The convention is to have a block with all the letters that start with a capital and sorted a-z, an empty line and then all the lowercase.

```javascript
var PixelRatio = require('PixelRatio');
var StyleSheetRegistry = require('StyleSheetRegistry');
var StyleSheetRegistry = require('StyleSheetRegistry');

var flatten = require('flattenStyle');
```